overlord/configstate/configcore/handlers.go

```diff
@@ -146,7 +161,14 @@ func filesystemOnlyApply(rootDir string, cfg config.ConfGetter, opts *sysconfig.
 		return fmt.Errorf("internal error: root directory for configcore.FilesystemOnlyApply() not set")
 	}
 
-	ctx := &fsOnlyContext{RootDir: rootDir}
+	if opts == nil {
+		opts = &sysconfig.FilesystemOnlyApplyOptions{}
+	}
+
+	ctx := &fsOnlyContext{
+		RootDir: rootDir,
+		UC20:    opts.UC20,
```

Comment:
so because what I explained is not clear to me that the flag is meaningful here?